src/XenCert/StorageHandlerUtil.py

```diff
@@ -716,7 +702,7 @@ def parse_config(vendor, product):
             re_product = re.compile(attr_map['product'].strip('"'))
             if (re_vendor.search(vendor) and re_product.search(product)):
                 xencert_print("matched vendor and product")
-                device_config = dict(multiPathDefaultsMap.items() + attr_map.items())
+                device_config = dict(d["defaults"] + attr_map.items())
```

Comment:
just curious, is the deleted multiPathDefaultsMap the same as the d["default"]?

Reply:
Not same, d["default"] is the correct setting parsed from multiple config, while multiPathDefaultsMap is hardcode in XenCert it's useless and only for logging.

Reply:
Indeed, the set read from the system is more complete and actually reflects what the system is applying rather than some (potentially out of date) hardcoded defaults in the cert kit.